README.md

```diff
@@ -174,7 +174,14 @@ ui5lint --fix
 ```
 
 Currently, issues of the following rules are fixable:
-- no-globals: Usage of globals that are part of UI5 (e.g. `sap.m.Button`) are replaced with the corresponding module import
+- no-globals: Usage of globals that are part of UI5 (e.g. `sap.m.Button`) are replaced with the corresponding module import (in JS files)
+- no-deprecated-api: Usage of deprecated APIs
+  - Usages of deprecated APIs on `sap.ui.getCore().getConfiguration()` a.k.a. "Configuration Facade" are replaced with the corresponding recommended APIs (in JS files)
```

Comment:
Maybe we should make it more clear that only some deprecated API can be replaced?

```suggestion
  - Usages of some deprecated APIs on `sap.ui.getCore().getConfiguration()` a.k.a. "Configuration Facade" are replaced with the corresponding recommended APIs (in JS files)
```